pkg/controllers/hub/trafficmanagerbackend/controller_integration_test.go

```diff
@@ -108,8 +129,31 @@ var _ = Describe("Test TrafficManagerBackend Controller", func() {
 			Expect(k8sClient.Create(ctx, backend)).Should(Succeed())
 		})
 
+		It("Updating TrafficManagerProfile status to accepted true and it should trigger controller", func() {
+			By("By updating TrafficManagerProfile status")
```

Comment:
Without adding this condition status, what's gonna happen? The trafficManagerProfile will fail right?

Reply:
added the validation, the condition will be set to unknown